apps/backoffice-v2/src/pages/Entities/hooks/useEntities/useEntities.tsx

```diff
@@ -86,20 +78,27 @@ export const useEntities = () => {
 
   const { workflowDefinition } = useCaseCreationWorkflowDefinition();
 
+  const isNoCases = !isLoading && Array.isArray(cases) && !cases.length;
```

Comment:
Do you think we should use `useMemo` here?

Reply:
`useMemo` is never used for derived state, maybe something very rare.